src/screens/Search/FilterSub.js

```diff
@@ -28,6 +28,7 @@ class FilterSub extends Component {
     this.renderItem = this.renderItem.bind(this);
     this.renderLength = this.renderLength.bind(this);
     this.renderFooter = this.renderFooter.bind(this);
+    this.renderSectionHeader = this.renderSectionHeader.bind(this);
```

Comment:
All of these `.bind(this)` calls are exactly what that `whatever = () => {` style declarations are about, so you can actually get rid of all of these lines here and just swap the `onSubmit`, `renderSort`, `renderItem`, `renderLength`, `renderFooter`, and `renderSectionHeader` methods to be declared in that style and you can then get rid of the `constructor` entirely.